images/mariadb/docker-entrypoint.sh

```diff
@@ -0,0 +1,101 @@
+#!/usr/bin/env bash
+
+set -e
+set -x
```

Comment:
can you add here the 
```
set -eo pipefail
shopt -s nullglob
```
from the original docker-entrypoint?

Reply:
shopt is unsupported.